v2.0/import.md

```diff
@@ -65,6 +65,12 @@ Imported tables are treated as new tables, so you must [`GRANT`](grant.html) pri
 
 All nodes are used during tabular data conversion into key-value data, which means all nodes' CPU and RAM will be partially consumed by the [`IMPORT`](import.html) task in addition to serving normal traffic.
 
+## Viewing and Controlling Import Jobs
+
+Whenever you initiate an import, CockroachDB registers it as a job, which you can view with [`SHOW JOBS`](show-jobs.html).
+
+After the import has been initiated, you can control it with [`PAUSE JOB`](pause-job.html), [`RESUME JOB`](resume-job.html), and [`CANCEL JOB`](cancel-job.html).
```

Comment:
We may want to add another sentence here like "IMPORT cannot reuse its existing work, so issuing a `PAUSE` then `RESUME` will cause it to restart from the beginning."